packages/manager/apps/pci-instances/src/adapters/tanstack/instances/right/mapper.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  TOperationStatus,
+  TInstancreCreationData,
+} from '@/domain/entities/instance';
+import {
+  TCreateInstancePayloadDTO,
+  TCreateInstanceResponseDTO,
+  TOperationStatusDTO,
+} from './dto.type';
+
+type TCreateInstancePayload = {
```

Comment:
Could be moved to ten entity folder no? As it's your form (front entity) 


```suggestion
type TCreateInstanceEntity = {
```